liota/core/package_manager.py

```diff
@@ -376,37 +359,67 @@ def run(self):
 
             # Switch on message content (command), determine what to do
             command = msg[0]
-            if command in ["load", "unload", "delete", "reload", "update"]:
+            if command in ["load", "reload", "update"]:
                 #-----------------------------------------------------------
                 # Use these commands to handle package management tasks
 
                 with package_lock:
                     if len(msg) < 2:
                         log.warning("No package is specified: %s" % command)
                         continue
-                    if len(msg) > 2:
-                        list_packages = msg[1:]
+                    if len(msg) > 3:
```

Comment:
I checked ==2 case, after >3, have you seen?
                    if len(msg) == 2:
                        log.warning("No checksum of {0} is specified: {1}".
                                    format(file_name, command))